analysis_options.yaml

```diff
@@ -20,10 +20,12 @@ linter:
     - directives_ordering
     - only_throw_errors
     - package_api_docs
+    - prefer_const_declarations
+    - prefer_final_locals
```

Comment:
Just slid that one in there, did ya?

Reply:
Oh, lol it's just sorted. I was like, "how is this CL not much much bigger???"

Reply:
I did add `prefer_const_declarations' 😁